src/helix-ui/styles/elements/hx-file-tile.less

```diff
@@ -1,19 +1,6 @@
 hx-file-tile {
   background-color: @gray-50;
-  color: @gray-900;
   display: block;
-  font-size: 0.875rem; //14px
-  min-width: 11rem; //176px
-
-  &[invalid] {
-    border: 2px solid @red-900;
-  }
-
-  > hx-progress {
-    margin-top: 0.5rem;
-  }
-}
-
-.hxCol > hx-file-tile {
-  min-width: none;
+  height: 3.75rem; // NOT: taller than spec
```

Comment:
FYI @ design: This is slightly taller than spec, to account for content shift in/out of invalid state.

Reply:
Also FYI: this was changed because of a visual difference in content alignment when transitioning between invalid and non-invalid states